RecoTracker/DeDx/plugins/DeDxHitCalibrator.cc

```diff
@@ -0,0 +1,399 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackDeDxHits.h"
+#include "DataFormats/TrackReco/interface/DeDxHit.h"
+#include "DataFormats/TrackReco/interface/DeDxHitInfo.h"
+#include "CondFormats/PhysicsToolsObjects/interface/DeDxCalibration.h"
+#include "CondFormats/DataRecord/interface/DeDxCalibrationRcd.h"
+
+#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
+#include "Geometry/CommonDetUnit/interface/PixelGeomDetUnit.h"
+#include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
+#include "Geometry/TrackerGeometryBuilder/interface/StripGeomDetUnit.h"
+#include "CalibTracker/SiPixelESProducers/interface/SiPixelGainCalibrationOfflineService.h"
+
+#include <fstream>
+
+class DeDxHitCalibrator : public edm::stream::EDProducer<> {
+public:
+  enum { isNormal, isBelow, isOver };
```

Comment:
```suggestion
  enum { kIsNormal, kIsBelow, kIsOver };
```
http://cms-sw.github.io/cms_coding_rules.html#2--naming-rules 2.17